test/common/upstream/health_check_fuzz.h

```diff
@@ -1,4 +1,6 @@
 #pragma once
+#include <bits/stdint-uintn.h>
```

Comment:
huh, did you need this header?

Reply:
No, deleting now I don't know where this came from.